apps/style/code-studio/pd.scss

```diff
@@ -42,6 +42,22 @@
       margin: 0;
     }
   }
+
+  table.table.table-auto-width {
```

Comment:
As it stands, it only applies to tables in studio that have the classes `table` and `table-auto-width`. I made up that last one for this purpose. It seems pretty well scoped to me. Nothing else uses this new class, `table-auto-width`, and the name is descriptive. Also this way it can be used elsewhere if we want it, by applying that class.

Do you have a suggestion to scope it further?